compiler/rustc_const_eval/src/interpret/step.rs

```diff
@@ -207,7 +207,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 }
             }
 
-            Repeat(ref operand, _) => {
+            Repeat(ref operand, _, _) => {
```

Comment:
If there is possible UB due to that boolean being wrong, then the interpreter must be adjusted to be able to detect that UB.